tests/baselines/reference/for-of32.errors.txt

```diff
@@ -0,0 +1,7 @@
+tests/cases/conformance/es6/for-ofStatements/for-of32.ts(1,10): error TS7022: 'v' implicitly has type 'any' because it is does not have a type annotation and is referenced directly or indirectly in its own initializer.
+
+
+==== tests/cases/conformance/es6/for-ofStatements/for-of32.ts (1 errors) ====
+    for (var v of v) { }

```

Comment:
Good catch. I'll try it out, thanks! I'll also test the three bullets you mentioned above, and see what happens. It is possible they will have to be fixed more holistically with regular variable declarations if they are broken.